alica_engine/include/engine/AlicaContext.h

```diff
@@ -29,7 +29,9 @@ namespace alica
 
```

Comment:
stl headers should be coming in before project headers. maybe this got changed in json_plan_designer branch? 
the order should generally be:
- (<file>.h for the same <file>.cpp)
- C++ standard library headers
- dependent libraries' headers
- other project headers.

Reply:
Conclusion: we stick to the engines code guideline (system headers go last)